common/arch/sdl/window.cpp

```diff
@@ -25,18 +25,18 @@ namespace dcx {
 static window *FrontWindow = NULL;
 static window *FirstWindow = NULL;
 
-window *window_create(grs_canvas *src, int x, int y, int w, int h, window_subfunction<void> event_callback, void *data, const void *createdata)
+window::window(grs_canvas *src, int x, int y, int w, int h, window_subfunction<void> event_callback, void *data, const void *createdata)
```

Comment:
Yes I considered that, but didn't this time around to minimise changes. I would be happy to do this.